apidef/oas/security_test.go

```diff
@@ -512,11 +512,11 @@ func TestOAS_GoPlugin(t *testing.T) {
 	}
 
 	var api apidef.APIDefinition
-	oas.extractGoPluginTo(&api)
+	oas.getTykAuthentication().ExtractTo(&api)
 
 	var convertedOAS OAS
 	convertedOAS.SetTykExtension(&XTykAPIGateway{Server: Server{Authentication: &Authentication{}}})
-	convertedOAS.fillGoPlugin(api)
+	convertedOAS.getTykAuthentication().Fill(api)
```

Comment:
Do we need nil checks for getTykAuthentication here, I see a nil check higher up

Reply:
In the line above, it is initialized: 
```
convertedOAS.SetTykExtension(&XTykAPIGateway{Server: Server{Authentication: &Authentication{}}})
```